packages/client/src/generation/TSClient/TSClient.ts

```diff
@@ -144,7 +144,7 @@ ${buildNFTAnnotations(engineType, platforms, relativeOutdir)}
     const collector = new ExportCollector()
 
     const commonCode = commonCodeTS(this.options)
-    const models = Object.values(this.dmmf.modelMap).reduce((acc, model) => {
+    const models = Object.values(this.dmmf.typeModelMap).reduce((acc, model) => {
```

Comment:
Maybe could be better if we also rename the const? It can be confusing now I think.
From `const models` to `const modelsType` maybe?

Reply:
What makes more sense to you, that looks good. Better than keeping the old const name I think